Userland/Libraries/LibPDF/Renderer.cpp

```diff
@@ -593,7 +593,39 @@ RENDERER_TODO(shade)
 RENDERER_TODO(inline_image_begin)
 RENDERER_TODO(inline_image_begin_data)
 RENDERER_TODO(inline_image_end)
-RENDERER_TODO(paint_xobject)
+RENDERER_HANDLER(paint_xobject)
+{
+    VERIFY(args.size() > 0);
+    auto resources = extra_resources.value_or(m_page.resources);
+    auto xobject_name = args[0].get<NonnullRefPtr<Object>>()->cast<NameObject>()->name();
+    auto xobjects_dict = MUST(resources->get_dict(m_document, CommonNames::XObject));
+    auto xobject = MUST(xobjects_dict->get_stream(m_document, xobject_name));
+    dbgln("Handling xobject: {}", xobject);
```

Comment:
I think it would be good to hide those ´dbgln´s behind a debug macro and use ´dbgln_if´

Reply:
Sorry, that wasn't meant to go in in the first place! I'll simply remove it

Reply:
OK, all unnecessary dbgln removed now